### ISSUE TITLE
improve AnkiDroid accessiblity

### ISSUE THREAD

**Issue by tyuichis** [tyuichis/modern-anki-card-template#6]
## Google Accessibility Scanner
AnkiDroid light theme may need to be modified, as subject labels are failing contrast tests

<img src="https://github.com/user-attachments/assets/cf2d8d9f-0070-4b83-ab17-140af2248677" alt="light_theme_accessibility_scan" width="300px"/>

AnkiDroid dark theme seems to pass most tests.

✅ No typography changes necessary, thankfully.
✅ Content is properly structured, no changes necessary.

The label colors can be found here.

https://github.com/tyuichis/modern-anki-card-template/blob/main/data/_labelColors.json

### What needs to be done:
- Identify which labels are problematic in Light Mode (label colors are dependant on subject; must use 7 different subjects to target all 7 colors)

### Possible fixes:
- After identifying problematic labels, change the backdrop color of AnkiDroid light theme to enhance accessibility.

**Comment by tyuichis** [comment]
closed as contrast was reviewed with APCA:

Added APCA safe accent colors (45a41b2)
Enable High Contrast mode via your OS's settings to use these new APCA colors
By default, WCAG 2.0 colors are used.
All other text colors now pass APCA Lc -+90 contrast value (09b6e0c)
Icon colors are also tweaked for better contrast (3c9a1b9)
